build/__snapshots__/readme-parser.test.ts.snap

```diff
@@ -0,0 +1,1137 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
Snapshots are a bit annoying, but I think in the past we had at least one instance of nuking the options page due to some parser mistakes.

This can also help as to double-check what is actually being extracted from the readme. Currently the only way is to check the generated bundles or opening the options page (where screenshot URLs are not immediately visible for example)

On a related note, I wonder if we should export this list to a JSON file into the repo so that it can be used.

https://vitest.dev/api/expect.html#tomatchfilesnapshot